README.md

```diff
@@ -0,0 +1,61 @@
+# urdf-packager
+
+A small util to turn a robot model into a single self-contained archive.
+
+## Use
+
+### Env setup
+
+Create a ROS 2 ready environment:
+
+```bash
+lxc launch ubuntu:22.04 urdf-exporter --config=user.user-data="$(curl -L https://gist.githubusercontent.com/artivis/1fcfcc38f1cb8c087e27ebda5484f2ef/raw/ros-bare.cloud-init.yaml)"
+```
+
+Shell inside the environment as the `ubuntu` user:
+
+```bash
+lxc exec urdf-exporter --mode interactive -- /bin/sh -xac $@ubuntu - exec /bin/login -p -f
+```
+
+Create a workspace and clone the project:
+
+```bash
+mkdir -p ~/workspace/src
+cd ~/workspace/src
+git clone https://github.com/ros-navigation/nav2_minimal_turtlebot_simulation.git
+```
+
+Install the dependencies:
+
+```bash
+source /opt/ros/*/setup.bash
+rosdep install --from-path . -y -i
+```
+
+```bash
+apt install python3-pip
+python3 -m pip install resolve_robotics_uri_py xacro
```

Comment:
can't all that be mentioned in the `pyproject.toml`? This way `pip install .` would do everything

Reply:
`resolve_robotics_uri_py` is in the `pyproject`. 
However the script doesn't depend directly on `xacro`. It is the overall workflow that uses it.